shared/common-adapters/index.js

```diff
@@ -12,6 +12,7 @@ export {default as ClickableBox} from './clickable-box'
 export {default as Confirm} from './confirm'
 export {default as CopyText} from './copy-text'
 export {default as CopyableText} from './copyable-text'
+export {default as DesktopStyle} from './desktop-style'
```

Comment:
So we can insert css styles and have it be a no-op on mobile. Useful when you want to make css hover styles that don't matter in Mobile.